lib/config.js

```diff
@@ -7,7 +7,10 @@ var config = require('bedrock').config;
 var path = require('path');
 require('bedrock-validation');
 
-config['key-http'] = {};
+// var basePath = '/keys';
+// config['key-http'] = {};
+// config['key-http'].routes = {};
+// config['key-http'].routes.basePath = basePath;
```

Comment:
bedrock-key-http's main.js doesn't use the config in this way... perhaps it needs to be updated?  Anyway, the basePath gets set here based on other bedrock config, not the config.js file.  I believe these were commented out because they weren't used.  

Here's where basePath gets set in main:  https://github.com/digitalbazaar/bedrock-key-http/blob/master/lib/main.js#L34
Here's where that configuration gets set in bedrock-key:  https://github.com/digitalbazaar/bedrock-key/blob/testFramework/lib/config.js#L13

If we are trying to standardize how we do this, we can update main.js.  For example, the path in bedrock-agreement-http gets set using it's own config like the one commented out above here:  https://github.com/digitalbazaar/bedrock-agreement-http/blob/master/lib/index.js#L24

You or Matt might have more insight on this.